pkg/nvkind/cluster.go

```diff
@@ -300,3 +347,31 @@ func getConfigBytesFromExistingCluster(name string) ([]byte, error) {
 
 	return []byte(configMap.Data["config"]), nil
 }
+
+func runChildProcess(args []string, input string) error {
+	klog.Infof("nvkind: run: %v", args)
+	cmd := exec.Command(args[0], args[1:]...)
+
+	// Let child's stdout/err flow directly to current process's standard streams.
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+
+	// Write data to child's stdin.
+	if len(input) > 0 {
+		pipe, err := cmd.StdinPipe()
+		if err != nil {
+			return fmt.Errorf("attaching pipe to child: %w", err)
+		}
+
+		go func() {
+			defer pipe.Close()
+			io.WriteString(pipe, input)
+		}()
+	}
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("nvkind: err running %v: %w", args, err)
```

Comment:
```suggestion
		return fmt.Errorf("error running '%v': %w", args, err)
```

Reply:
:+1: also using `strings.Join()` here and not `%v` anymore